README.md

```diff
@@ -2,10 +2,61 @@
 |-|
 |This library is currently in development and has not undergone a thorough correctness assessment or security audit. It is therefore NOT suitable for production use until full approval is granted by the ANZ open source committee and this notice removed.|
 
-# decimal
+# Decimal
 
-This library implements fixed-precision decimal numbers based on IEEE 754
-decimal64.
+This library implements fixed-precision decimal numbers based on IEEE 754 decimal64.
 
-The goal is to fully support decimal64 and decimal128. Currently only supports
-decimal64, excluding subnormals, and needs more thorough testing.
+The goal is to fully support decimal64 and decimal128. Currently only supports decimal64, excluding subnormals, and needs more thorough testing.
+
+## Preliminary learning
+
+To gain familiarity with this project, it is reccomended to develop a further understanding of the floating point standard, below are some links to get you started
+- [IEEE 754-2008 revision - Wikipedia](https://en.wikipedia.org/wiki/IEEE_754-2008_revision)
+
+- [IEEE Standard for Floating-Point
+Arithmetic - PDF](http://www.dsc.ufcg.edu.br/~cnum/modulos/Modulo2/IEEE754_2008.pdf)
+
+- [Decimal specification from IBM](http://speleotrove.com/decimal/)
+
+## Running unit tests
+
+### IBM unit tests
+
+Our unit tests are provided by Mike Cowlishaw from IBM, they are referenced in our test suite `decimalSuite_test.go`
```

Comment:
s/, they/and/

Reply:
What does that mean?

Reply:
Josh has said it means `sub they with and` haha